docs/cpp/configure-intellisense-crosscompilation.md

```diff
@@ -0,0 +1,104 @@
+- The path to library headers for the system and project need to be retrieved from the target compiler, or manually specified.
+
+Sometimes this information can be obtained automatically and sometimes it is provided by an extension. In all cases, you can enter it manually if needed. If you have trouble, feel free to file an issue in the [VS Code documentation repository](https://github.com/Microsoft/vscode-docs/issues).
+
+## Install target SDK
+
+When you target an architecture that is different from the host development platform, the compiler and/or system libraries for the target architecture may need to be installed on the host development platform so that IntelliSense can reference them.
+
+The Microsoft Visual C++ components are included when you install Visual Studio.
+If you are using the MSVC compiler from a non-Windows platform, install the [Windows SDK](https://developer.microsoft.com/windows/downloads/windows-10-sdk).
+
+## Edit c_cpp_properties.json
+
+C/C++ extension configuration information is stored in the `c_cpp_properties.json` file. This file stores IntelliSense configuration settings per workspace and is located under `${workspaceFolder}/.vscode` where  the `tasks.json` and `launch.json` files are also saved.
+
+To create and enter configuration information in this file, select **C/C++: Edit Configurations (UI)** from the Command Palette (`kb(workbench.action.showCommands)`).
+
+![Command Palette](images/cpp/command-palette.png)
+
+This opens the **C/C++ Configurations** page. When you make changes here, VS Code writes them to the `c_cpp_properties.json` file in the `.vscode` folder.
+
+![Intellisense configuration window](images/wsl/intellisense-configurations-wsl.png)
+
+Values in `c_cpp_properties.json` take precedence over the `c_cpp_properties` set in the `settings.json` file.  If there are `c_cpp_properties` set in `settings.json` and the same properties are not set or does not exist in a `c_cpp_properties.json` file, then the extension will use the values in `settings.json`.
+
+If you are curious, you can open the `.vscode/c_cpp_properties.json` file directly. It'll look something like this:
+
+```json
+{
+    "configurations": [
+        {
+            "name": "Linux",
+            "includePath": [
+                "${workspaceFolder}/**"
+            ],
+            "defines": [],
+            "compilerPath": "/usr/bin/gcc",
+            "cStandard": "c11",
+            "cppStandard": "c++17",
+            "IntelliSenseMode": "clang-x64"
+        }
+    ],
+    "version": 4
+}
+```
+
+You can also edit the file directly by opening it from the .vscode subdirectory in your working directory, or by selecting **C/C++: Edit Configurations (JSON)** from the Command Palette (`kb(workbench.action.showCommands)`).
+
+## Set cross-compilation configuration settings
+
+The following settings need to be configured for intellisense to work when you are cross-compiling:
```

Comment:
Instead of framing it as "cross-compiling", say something like:

The following settings need to be configured for IntelliSense to work when you are working with a project that will be compiled for an architecture that is different from your host machine.

Reply:
Changed to: Configure the following settings so that IntelliSense will work when you are compiling for a different architecture than your host development machine.